3rdPartyLicense/LICENSE_Date_Nagi

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Small comment: `thirdparty/LICENSES/...` folder would be my suggestion for the folder containing all third party licenses

Reply:
Done in next push.